ui/lib/core/addon/utils/search-select-has-many.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Util to add/remove models in a hasMany relationship via the search-select component
+ *
+ * @example of using the util within an action in a component
+ *```js
+ * @action
+ * async onSearchSelectChange(selectedIds) {
+ *    const methods = await this.args.model.mfa_methods;
+ *   handleHasManySelection(selectedIds, methods, this.store, 'mfa-method');
+ *  }
+ *
+ * @param selectedIds array of selected options from search-select component
+ * @param modelCollection array-like, list of models from the hasMany relationship
```

Comment:
Is there any reason you're not using the convention for documentation of the `@param {Array} modelCollection - Array-like, list of ...`

Reply:
Since this isn't a component - I just pulled the jsdoc syntax so provide some explanation of what's going on since it's a newer pattern. 

I wasn't sure how folks would feel about it. I can commit to 100% jsdoc syntax if this documentation is helpful! But wanted feedback first before committing.

Reply:
I reads a little confusing without the of separation from the name to the description, which is why I thought of this. For example: @param store the store so we can call peekRecord(). 

I think my brain is used to documentation style we use for components, so it took a moment to adjust. My preference, which isn't a strong one, is to follow the styling we use for documentation in components.